app/controllers/api/v1/coaches_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Api::V1::CoachesController < ApplicationController
+  def index
+    coaches = Coach.all
```

Comment:
Here maybe for just the general convention, you can declare the variables with the prefix @. 
let coaches be @coaches